### ISSUE TITLE
As a developer, I want a AzDO build pipeline triggered for peer code review requests/updates

### ISSUE THREAD

**Issue by erikschlegel** [Microsoft/cobalt#54]
### Description
*As a developer, I want a AzDO build pipeline triggered for all PR requests/commits.*

This feature is intended to add a new azure devops build pipeline that's linked to a definition yml file from the `/devops/providers/templates/azure-pipelines.yml` in the cobalt repo. The new build definition should be created in the cobalt azure devops instance [here](https://dev.azure.com/csedallascrew/project-cobalt). This build pipeline should be triggered from newly submitted PR's and/or new commits to existing PR's. 

We should also add the necessary folder filters to avoid running builds for documentation changes. 

### Acceptance Criteria
* [x] Should assert that the checked in azure devops pipeline definition successfully passes in the Azure DevOps tool for newly posted PR's and new PR commits. 
* [x] Should assert that the build definition is running on an Ubuntu VM Image.
* [x] Should include a screenshot of the passed build. 
* [x] Should assert there's a mocked inline script step with a sample echo message. This will be replaced once the user story for adding the unit test step is complete. 

*Technical Design Document*
* [x] Add a CI devops flow diagram scoped to the integration environment within the `README.md`
* [x] Fill in the README in the azure-devops folder to describe the intent / usage of the build pipeline. We should also include step-by-step instructions on how to setup a new Azure DevOPS build pipeline as well as the GitHub repo connection. 
* [x] Add the azure devops build status badge to the `README` in the root. 

Also, here are a few points that need to be addressed:

1. This feature is dependent on a azure devops service connection linked to the github cobalt repo.

### Resources
Please include a screen shot 

### Tasks
- Add azure devops pipeline definition
- Add/update `README` documentation in both the root and azure-devops folders.
